### PR TITLE
Korean translation for Jstris

### PR DESCRIPTION
**Pull request by Ecdldaiiere**

### CODE REVIEW

File: kr/web.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    'longestGame' => '가장 긴 게임',
+    'totalB2B' => '모든 백투백',
+    'mostSent' => '가장 많이 보낸 공격',
+    'hours' => '시간',
+    'minutes' => '분',
+    
+    /* Stats */
+    'improvementStats' => '더 많은 내용',
+    //keep ':이름'
+    'improvementInfo' => '라인레이스 시간 (Y 축) vs 게임 한 날짜 (X 축). 강조된 선은 이름 또는 개인 기록을 표시합니다. 커서를 통해 임의대로 세부 정보를 확인 할 수 있습니다.',
```
Comment (jezevec10) [inline kr/web.php]: This line should contain `:name` place holder. This gets then replaced with player's name (if you look how this line is used on the site you will see your username mentioned in it).

The comment above should say `//keep ':name'`, this string has a special purpose and is not translated.

**Review by jezevec10 — CHANGES_REQUESTED** [review]
Thanks for the translation. There is a tiny mistake in the translation at the line I marked, please review it.

File: kr/web.php (hunk 1 above)
Comment (Ecdldaiiere) [reply kr/web.php]: ah okay. I fixed it back to //keep ':name'  do i have to issue another pull request or is it automatically updated?

Comment (jezevec10) [reply kr/web.php]: The translation itself at the line 74 should contain `:name`. It's where the username gets inserted, the comment is there just so it doesn't get deleted in editing.

After you make the change the pull request is updated automatically, no need for another one.

**Review by jezevec10 — APPROVED** [review]

**Pull request merged**